### PR TITLE
chore(Dependencies): upgrade dependencies: acceptance-testing, ts-to-flow

### PR DESCRIPTION
**Pull request by pladaria**

### CODE REVIEW

**Review by atabel — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,18 +9,13 @@ on:
...
         steps:
             - uses: actions/checkout@v2
               with:
                   token: ${{ secrets.FLOW_GITHUB_TOKEN }}
 
-            - name: Setup Node.js
-              uses: actions/setup-node@v1
-              with:
-                  node-version: 16
-
```
Comment (pladaria) [inline .github/workflows/release.yml:23]: using the node from the docker

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -11,25 +11,19 @@ on:
...
+        container: telefonicacdo/web-builder:chromium93.0.4577-1.2.3
 
         steps:
             - uses: actions/checkout@v2
 
-            - name: Use Node.js ${{ matrix.node-version }}
-              uses: actions/setup-node@v1
-              with:
-                  node-version: ${{ matrix.node-version }}
-
```
Comment (pladaria) [inline .github/workflows/ci.yml:28]: using the node from the docker

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -11,25 +11,17 @@ on:
...
             - name: Install dependencies
               run: yarn install --immutable --immutable-cache
 
-            - run: yarn ci
+            - name: Run tests
```
Comment (atabel) [inline .github/workflows/ci.yml:23]: well, not only tests, but ok

File: .github/workflows/deploy-master.yml [hunk 4]
```diff
@@ -4,10 +4,15 @@ on:
...
-        runs-on: self-hosted-novum
-        container: docker.tuenti.io/service-inf/web-builder:pptr10.4-1.0.2
+        runs-on: ubuntu-latest
+        container: telefonicacdo/web-builder:chromium93.0.4577-1.2.3
         steps:
             - uses: actions/checkout@v2
+
+            - name: install-vercel
+              # workaround for https://github.com/vercel/vercel/issues/6811
+              run: npm i -g vercel@canary
```
Comment (atabel) [inline .github/workflows/deploy-master.yml:14]: is this a temporal fix? can you create a ticket to revert it?

File: package.json [hunk 5]
```diff
@@ -181,6 +181,7 @@
     },
     "resolutions": {
         "react": "17.0.2",
-        "react-dom": "17.0.2"
+        "react-dom": "17.0.2",
+        "jest-resolve@26.6.2": "patch:jest-resolve@npm:26.6.2#.yarn/patches/jest-resolve-npm-26.6.2-6ad3c87f85"
```
Comment (atabel) [inline package.json:185]: revert this

File: .github/workflows/ci.yml (hunk 3 above)
Comment (pladaria) [reply .github/workflows/ci.yml:23]: yes.. this is more like "Run everything" XD

File: .github/workflows/deploy-master.yml (hunk 4 above)
Comment (pladaria) [reply .github/workflows/deploy-master.yml:14]: https://jira.tid.es/browse/WEB-417

**Pull request merged**